src/aioswitcher/device/tools.py

```diff
@@ -165,6 +166,34 @@ def convert_str_to_devicetype(device_type: str) -> DeviceType:
     return DeviceType.MINI
 
 
+def convert_token_to_packet(
```

Comment:
I'm just making sure we aren't missing anything. Currently, no one is using this function, only tests. This is meant to be used with future steps of this work, right?

Reply:
Correct. In the next PR it will be used